project/pom.xml

```
@@ -26,6 +26,7 @@
         <maven.compiler.target>16</maven.compiler.target>
         <!-- Default file encoding -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.target.sourceEncoding>UTF-8</project.target.sourceEncoding>
```

Тут ошибка. Должно быть 
```suggestion
        <project.build.targetEncoding>UTF-8</project.target.sourceEncoding>
```

---

done